crates/core/app/tests/mock_consensus.rs

```diff
@@ -37,6 +40,32 @@ async fn mock_consensus_can_send_an_init_chain_request() -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Exercises that the mock consensus engine can provide a single genesis validator.
+#[tokio::test]
+async fn mock_consensus_can_define_a_genesis_validator() -> anyhow::Result<()> {
+    // Install a test logger, acquire some temporary storage, and start the test node.
+    let guard = common::set_tracing_subscriber();
+    let storage = TempStorage::new().await?;
+    let test_node = common::start_test_node(&storage).await?;
+
+    let identity_key = test_node.penumbra_identity_key();
+    match storage
+        .latest_snapshot()
+        .get_validator_state(&identity_key)
+        .await?
+        .ok_or_else(|| anyhow!("genesis validator state was not found"))?
+    {
+        penumbra_stake::validator::State::Active => {}
+        other => panic!("unexpected genesis validator state, found: {other}"),
+    }
```

Comment:
small question for @erwanor: do genesis validators start in `Active`?

Reply:
I think we could make it always the case, it seems simplest. There's no reason to define a validator in the genesis file if it's not going to be immediately participating, after all.